backend/backend/src/main/java/de/aaaaaaah/velcom/backend/access/TaskWriteAccess.java

```diff
@@ -66,10 +66,10 @@ public void onTaskDelete(Consumer<TaskId> deleteHandler) {
 	 * @param tasks the tasks to insert
 	 */
 	public void insertTasks(Collection<Task> tasks) {
-		insertTasks(tasks, databaseStorage.acquireContext());
+		insertTasks(tasks, databaseStorage.acquireWriteAccess());
```

Comment:
Why is no try-with-resources used here?

Reply:
Fixed in https://github.com/IPDSnelting/velcom/commit/9c85694d224e55f310d9330460dca600ea140c58